.clang-format

```diff
@@ -0,0 +1,249 @@
+---
+Language:        Cpp
```

Comment:
Can't we just use basedonstyle: google?

Ex: https://github.com/googleapis/google-cloud-cpp/blob/main/.clang-format

Reply:
Sure thing! This is definitely better, didn't know it was possible.